### ISSUE TITLE
[Issue] Random Song not working

### ISSUE THREAD

**Issue by Teslahaha** [TechShivvy/spicetify-extensions#1]
Random Song doesn't work on my Spotify... I installed it through the Marketplace. Like the description said, it is supposed to show a shuffle button at the top left bar, but that bar doesn't exist. Therefore I can't use it. Could there be an alternative way to use this extension? Maybe through the context menu instead?

**Comment by TechShivvy** [comment]
@Teslahaha I don't believe placing it in the context menu (right-click/profile menu) is appropriate or logical. I'll try to add a keyboard shortcut instead. 
Also, It would be nice if the top bar issue could be resolved; unfortunately, I don't know how to fix it. I recommend joining the Spicetify Discord and raising a support token for assistance.
As there have been significant changes between Spotify and Spicetify on the backend, it's causing the extension to currently malfunction. I'll work on resolving this issue too asap.

**Comment by TechShivvy** [comment]
In the latest version ([release v1.2.0](https://github.com/TechShivvy/spicetify-extensions/tree/v1.2.0)), I have added a keyboard shortcut of <kbd> Alt + R </kbd>, also added an option in the right click menu when you right click on users.

Added in PR: #9 

cc: @Teslahaha
